docs/Hooks-Script.md

```diff
@@ -1 +1,210 @@
-# TODO
```

Comment:
We should keep file names strictly lower case as a standars as the wiki cannot see "apage" == "Apage" so better just have the same on all pages